### DISCUSSION TITLE
AlDente 1.28.5

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1346] [General]
### Pro

- Removed a warning that AlDente would incorrectly show on some specific power supplies.
- The battery info widget now only shows the time to full when the battery is currently charging.

### All

- Fixed an issue introduced with macOS 15 that caused the menu bar icon to not adapt correctly on unfocused monitors.

With peace & love from Vienna,
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.28.5'>AlDente 1.28.5</a>.</em>

**Comment by DamienBlackwood** [comment]


**Reply by MatthiasKerbl** [reply]
Hi @DamienBlackwood,

You can find the premium support email address in your purchasing emails.

Best,
Matthias

**Comment by DamienBlackwood** [comment]


**Reply by MatthiasKerbl** [reply]
Hi @DamienBlackwood,

you can request to get the purchasing email again by deactivating your license in AlDente and clicking on "Forgotten your license key?"
<img width="778" alt="Screenshot 2024-10-22 at 12 20 30" src="https://github.com/user-attachments/assets/805b5cf2-34bb-4cbe-91dd-baa02b5839a1">

Afterward, just enter your email address and you will receive the email again, including the support email address.

Best,
Matthias

**Comment by chaonai0091** [comment]
I have a problem. I already purchased AlDente Pro but didn’t receive any email. Could you please contact me?(chaonai009@gmail.com) @MatthiasKerbl @davidwernhart

**Reply by MatthiasKerbl** [reply]
Hi @chaonai0091,

I am sorry to hear that you encountered an issue during the purchase.
Please take a look at our [FAQ](https://apphousekitchen.com/faq/) for that case:

There are multiple reasons why you did not get your license code after the purchase:

Sometimes it takes up to 30 Minutes for the purchase emails to go through.
The purchase emails end up in the spam folder, please check it.
You had a typo in the email address during your purchase.
Failures of the payment networks/providers.
Please contact the support team of our merchant of record Paddle at [assist@paddle.com](mailto:assist@paddle.com) and include the email address you used to purchase AlDente Pro. Paddle support will try to solve your issue as quickly as possible.

Best,
Matthias

**Reply by chaonai0091** [reply]
It has been two days since I purchased AlDente and contacted assist@paddle.com. May I ask how long it typically takes for them to respond?

**Reply by MatthiasKerbl** [reply]
Hi @chaonai0091,

depending on different factors, it usually takes between 1-5 workdays.

Best,
Matthias
